src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Commands/Create/CreateDialogCommandValidator.cs

```diff
@@ -44,6 +44,10 @@ public CreateDialogCommandValidator(
                 .WithMessage($"'{{PropertyName}}' must be greater than or equal to '{nameof(CreateDialogCommand.CreatedAt)}'.")
                 .When(x => x.CreatedAt != default && x.UpdatedAt != default);
 
+        RuleFor(x => x.IdempotentKey)
+            .MaximumLength(36)
+            .WithMessage("'{{PropertyName}}' can't be longer than 36 characters.");
```

Comment:
```suggestion
        RuleFor(x => x.IdempotentKey)
            .MaximumLength(36);
```

De innebygde metodene til fluent validation har en error message innebygd. Du trenger ikke en custom en her.